.github/workflows/ci.yaml

```diff
@@ -58,3 +58,69 @@ jobs:
           python -m pip install .[dev-noxfile]
       - name: Run nox
         run: nox
+
+  create-github-release:
+    needs: ["protolint", "gen-docs", "tests", "build-dist"]
```

Comment:
Here you have the missing `build-dist` job as a dependency too (which is correct, you just forgot to add that job too).